modules/time.js

```diff
@@ -0,0 +1,11 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+function displayDate() {
```

Comment:
- [ ] Kind remainder to change this as well to the arrow function, That should be all changes 👍🏻